var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py

```diff
@@ -19,8 +19,8 @@ class PyMorphologyRepairWorkflow(PythonPackage):
 
     depends_on('py-setuptools', type=('build', 'run'))
 
-    depends_on('py-neuroc', type='run')
-    depends_on('py-morph-tool', type='run')
-    depends_on('py-neuror', type='run')
-    depends_on('py-more-itertools', type='run')
-    depends_on('py-morph-validator', type='run')
+    # depends_on('py-neuroc', type='run')
+    # depends_on('py-morph-tool', type='run')
+    # depends_on('py-neuror', type='run')
+    # depends_on('py-more-itertools', type='run')
+    # depends_on('py-morph-validator', type='run')
```

Comment:
can we add these again? or is this a future PR?

Reply:
This is for a future PR by @GianlucaFicarelli

Reply:
should be done quickly

Reply:
Also the current module is not working anyway from what I understood

Reply:
Ok, thank you for clarifying! :)